### PR TITLE
Reword “Create an External Load Balancer” task

### LINKED ISSUE #25195: Stale information about “future” Kubernetes in Create an External Load Balancer task
**This is a Bug Report**

**Problem:**
[Create an External Load Balancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/) mentions:
> No support for weights is provided for the 1.4 release, but may be added at a future date.

(see https://github.com/kubernetes/website/blame/8b174740e5d9aa01219542fc6f4cf5649965d43e/content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md#L199)

Weights aren't built in to Service / Endpoints / EndpointSlice / Ingress but you can indicate weighting with annotations and similar mechanisms. For example: https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/ingress/annotations/#traffic-routing.

**Proposed Solution:**

Remove the text:
> No support for weights is provided for the 1.4 release, but may be added at a future date.

Revise https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/ to explain that each implementation uses its own mechanisms for weighting and that these are not standardized.

**Page to Update:**
https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/

**Kubernetes Version:**
v1.19

**Additional Information:**
/language en
/sig network

### PR DESCRIPTION
**Pull request by sftim**
- general cleanup
- update sample output
- use more tooltips
- avoid specifying specific cloud providers
- remove dated content

Fixes #25195

[[preview](https://deploy-preview-27182--kubernetes-io-master-staging.netlify.app/docs/tasks/access-application-cluster/create-external-load-balancer/)]

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27182--kubernetes-io-master-staging.netlify.app/docs/tasks/access-application-cluster/create-external-load-balancer/

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 1]
```diff
@@ -86,64 +83,66 @@ information through `kubectl`:
...
-the service controller allocates a port from your cluster's NodePort range. You
-can configure that range by setting an API server command line option,
-`--service-node-port-range`. It will use the
-user-specified `healthCheckNodePort` value if specified by the client. It only has an
-effect when `type` is set to LoadBalancer and `externalTrafficPolicy` is set
-to Local.
-
-Setting `externalTrafficPolicy` to Local in the Service configuration file
-activates this feature.
+By default, the source IP seen in the target container is *not the originali
```
Comment (tengqm) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:120]: typo

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 2]
```diff
@@ -86,64 +83,66 @@ information through `kubectl`:
...
+source IP* of the client. To enable preservation of the client IP, the following
+fields can be configured in the `.spec` of the Service:
+
+* `.spec.externalTrafficPolicy` - denotes if this Service desires to route
+  external traffic to node-local or cluster-wide endpoints. There are two available
+  options: Cluster (default) and Local. Cluster obscures the client source
```
Comment (tengqm) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:126]: I'd suggest we use code style for `Cluster` and `Local` for clarity.

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 3]
```diff
@@ -155,7 +154,20 @@ spec:
   type: LoadBalancer
 ```
 
-## Garbage Collecting Load Balancers
+### Caveats and limitations when preserving source IPs
+
+Load balancing services from some cloud providers do not let you configure different weights for each target.
+
+With each target weighted equally in terms of sending traffic to Nodes, external traffic is not equally load balanced
+across different Pods. The external load balancer is unaware of the number of Pods on each node that is used as a
```
Comment (tengqm) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:162]: I suspect the last 'is' should be 'are'. If we use 'is', 'that' may refer to 'the number'.

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 4]
```diff
@@ -155,7 +154,20 @@ spec:
...
+target.
+
+Where _NumServicePods_ ≪  _NumNodes_ or _NumServicePods_ ≫  _NumNodes_, a fairly close-to-equal
```
Comment (tengqm) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:165]: Please do not use "≪". This symbol is not standard ASCII, and it means totally different things in Chinese.

Comment (sftim) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:165]: We don't restrict docs to US-ASCII (that would be awful). Good to know about ≪. I wonder if there's a way to make it clear this is a (Castesian-style) equation? 二 vs = wouldn't be mistaken in this context.

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:120]: Yes. originali 😄

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 4 above)
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:165]: Yes, this symbol is more difficult to read.

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 5]
```diff
@@ -86,64 +83,66 @@ information through `kubectl`:
...
+  to Local.
+
+Setting `externalTrafficPolicy` to Local in the Service manifest
+activates this feature. For example:
 
 ```yaml
 apiVersion: v1
-kind: Service
-metadata:
+kind: Service metadata:
```
Comment (oomichi) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:145]: I think the original seems good here.

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 5 above)
Comment (sftim) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md:145]: Oops. Good catch.

**Comment by tengqm** [comment]
Looks to me this one is ready.
/lgtm

**Comment by oomichi** [comment]
Thanks for updating.

/lgtm

**Comment by kbhawkey** [comment]
Thanks @sftim .
/approve

**Pull request merged**